src/objects/viewport.ts

```diff
@@ -85,7 +85,7 @@ export async function centerViewport({ animate }: CenterViewportParameters = {})
       x: x + width / 2,
       y: y + height / 2 - guidesOffset,
     },
-    scale,
+    scale: Math.min(scale, 1),
```

Comment:
This will stop showing in this diff once the [upstream PR](https://github.com/PrefectHQ/graphs/pull/321/files) is rebased.